NorthstarDLL/serverauthentication.cpp

```diff
@@ -146,7 +146,9 @@ bool ServerAuthenticationManager::VerifyPlayerName(const char* authToken, const
 		spdlog::info("Rejecting player with name '{}' because the name contains forbidden characters", name);
 		return false;
 	}
-
+	// TODO: We should really have a better way of doing this for singleplayer
+	// Best way of doing this would be to check if server is actually in singleplayer mode, or just running a SP map in multiplayer
+	// Currently there's not an easy way of checking this, so we just disable this check if mapname starts with `sp_`
```

Comment:
We should make it more clear that this completely removes player name checking.

Reply:
Resolved with abc203b39ccfec555f67857845b2823d97839161